<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MergeNotesDialog</class>
 <widget class="QDialog" name="MergeNotesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MergeNotesDialog.qrc">
    <normaloff>:/icons/resources/logo_320x320.png</normaloff>:/icons/resources/logo_320x320.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnUpList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Move note up</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="MergeNotesDialog.qrc">
           <normaloff>:/icons/resources/arrow_up.png</normaloff>:/icons/resources/arrow_up.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnDownList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Move note down</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="MergeNotesDialog.qrc">
           <normaloff>:/icons/resources/arrow_down.png</normaloff>:/icons/resources/arrow_down.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="QListWidget" name="notesList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>4</verstretch>
          </sizepolicy>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::IgnoreAction</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QToolBox" name="toolBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageNotes">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>578</width>
         <height>281</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Merged Notes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Title:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="mergedNoteTitle">
            <property name="placeholderText">
             <string>Enter a title for the merged note</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnMergeNotes">
          <property name="toolTip">
           <string>Merge notes</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="MergeNotesDialog.qrc">
            <normaloff>:/icons/resources/arrow_in.png</normaloff>:/icons/resources/arrow_in.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textMergedNotes"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Tags:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="mergedNoteTags"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pagePreview">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>578</width>
         <height>281</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Merged Preview</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QPushButton" name="btnExportHtml">
            <property name="toolTip">
             <string>Export to HTML</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="MergeNotesDialog.qrc">
              <normaloff>:/icons/resources/html.png</normaloff>:/icons/resources/html.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btnPrintMergedPreview">
            <property name="toolTip">
             <string>Print the merged notes</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="MergeNotesDialog.qrc">
              <normaloff>:/icons/resources/printer.png</normaloff>:/icons/resources/printer.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QTextBrowser" name="textMergedPreview"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="MergeNotesDialog.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MergeNotesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>582</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MergeNotesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnUpList</sender>
   <signal>clicked()</signal>
   <receiver>MergeNotesDialog</receiver>
   <slot>click_note_up()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>580</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDownList</sender>
   <signal>clicked()</signal>
   <receiver>MergeNotesDialog</receiver>
   <slot>click_note_down()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnExportHtml</sender>
   <signal>clicked()</signal>
   <receiver>MergeNotesDialog</receiver>
   <slot>click_export_html()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPrintMergedPreview</sender>
   <signal>clicked()</signal>
   <receiver>MergeNotesDialog</receiver>
   <slot>click_print()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMergeNotes</sender>
   <signal>clicked()</signal>
   <receiver>MergeNotesDialog</receiver>
   <slot>click_merge_notes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>596</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>click_note_up()</slot>
  <slot>click_note_down()</slot>
  <slot>click_export_html()</slot>
  <slot>click_print()</slot>
  <slot>click_merge_notes()</slot>
  <slot>drop_note()</slot>
 </slots>
</ui>
